src/controllers/sendResetToken.js

```diff
@@ -0,0 +1,114 @@
+import { randomBytes } from 'crypto';
+import models from '../db/models';
+import sendgrid from '../middlewares/helper';
+
+const { User } = models;
+
+/**
+  * @description class representing reset user password
+  *
+  * @class SendResettoken
+  */
+class SendResetToken {
+  /**
+    * @description - This method is responsible for creating creating password reset token
+    *
+    * @static
+    * @param {object} request - Request sent to the router
+    * @param {object} response - Response sent from the controller
+    *
+    * @returns {object} - object representing response message
+    *
+    */
+  static async resetToken(request, response) {
+    try {
+      const { email } = request.body;
+      if (email.trim().length < 1) {
```

Comment:
Nice job validating for an empty email field 👍🏿, but I feel like this validation could be abstracted, maybe as a middleware. What do you think?

Reply:
That would be great, but it's just a small line of code, so I thought it was not unnecessary.